zerver/lib/notifications.py

```diff
@@ -281,15 +282,32 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
 
     text_content = loader.render_to_string('zerver/missed_message_email.txt', template_payload)
     html_content = loader.render_to_string('zerver/missed_message_email.html', template_payload)
-
-    msg = EmailMultiAlternatives(subject, text_content, from_email, [user_profile.email],
-                                 headers = headers)
-    msg.attach_alternative(html_content, "text/html")
-    msg.send()
+    email_content = {
+        'subject': subject,
+        'text_content': text_content,
+        'html_content': html_content,
+        'from_email': from_email,
+        'to': [user_profile.email],
+        'headers': headers
+    }
+    queue_json_publish("missedmessage_email_senders", email_content, lambda email_content: None)
```

Comment:
I think that last lambda (which is what tests do when this code path is hit) should be `send_missedmessage_email` instead.